### PR TITLE
Add binaryiotypec

### PR DESCRIPTION
**Pull request by KlausPopp**

### CODE REVIEW

File: examples/binaryIotypeC/stream/main.go [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+			log.Fatalf("ReadStreamData failed: %v\n", err)
+		} else {
+			samples := sd.FSData.GetSamples()
+			log.Printf("got stream data seq=%d ts=%d samples=%d\n", sd.Sequence, sd.DeliveryTimestamp, len(samples))
+
+			for i, sample := range samples {
+				if firstTs == 0 {
+					firstTs = sample.Timestamp
+				}
+				log.Printf("sample %d: relTs=%10dus values=b%016b valid=b%016b", i, sample.Timestamp-firstTs, sample.Values, sample.ValueValid)
```
Comment (FranziskaKnoedtel) [inline examples/binaryIotypeC/stream/main.go:105]: Not mandatory, but it looks clearer:
`log.Printf("sample %d: relTs=%10dus values=b%0*b valid=b%0*b", i, sample.Timestamp-firstTs, numberOfChannels, sample.Values, numberOfChannels, sample.ValueValid)`

**Review by FranziskaKnoedtel — CHANGES_REQUESTED** [review]

File: examples/binaryIoTypeC/stream/main.go [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+		}
+		if time.Since(start) > time.Second*time.Duration(*runtime) {
+			break
+		}
+	}
+}
+
+func manipulateOutputs(c *binio.Client, numberOfChannels int, wg *sync.WaitGroup, quit chan bool) {
+	go func() {
+		var values uint32 = 0x00
```
Comment (FranziskaKnoedtel) [inline examples/binaryIoTypeC/stream/main.go:116]: What is this variable used for?

**Review by FranziskaKnoedtel — CHANGES_REQUESTED** [review]

File: examples/binaryIoTypeC/stream/main.go (hunk 2 above)
Comment (KlausPopp) [reply examples/binaryIoTypeC/stream/main.go:116]: its a bug

Comment (FranziskaKnoedtel) [reply examples/binaryIoTypeC/stream/main.go:116]: You can remove the lines commented out:
`func manipulateOutputs(c *binio.Client, numberOfChannels int, wg *sync.WaitGroup, quit chan bool) {
	go func() {
		//var values uint32 = 0x00
		i := uint32(0)
		//var direction int32 = 1
		chMask := uint32((1 << numberOfChannels) - 1)
		for {
			select {
			case <-quit:
				wg.Done()
				return
			default:
				//values += uint32(direction)
				err := c.SetAllOutputs(i, chMask)
				if err != nil {
					log.Printf("can't set outputs: %v", err)
				}
				time.Sleep(time.Millisecond * 200)
				i++
				/*if i%chMask == 0 {
					direction *= -1
				}*/
			}
		}
	}()
}`

**Review by FranziskaKnoedtel — CHANGES_REQUESTED** [review]

**Pull request merged**